File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution{
 hasDuplicate(nums) {
  for (let i = 0; i < nums.length; i++) {
    for (let x = i + 1; x < nums.length; x++) {
      if (nums[x] === nums[i]) {
        return true;
      }
    }
  }
  return false;
}
}
    

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include <unordered_set>
#include <iostream>

class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> my_set; 
        for(int my_num : nums){
            if(my_set.contains(my_num)==true){
                return true;
            }else{
                my_set.insert(my_num);
            }
        }
        return false;
    }
};